examples/gh-pages/package.json

```diff
@@ -6,7 +6,7 @@
     "build": "next build",
     "start": "next start",
     "export": "next export",
-    "deploy": "rm -rf node_modules/.cache && next build && next export && touch out/.nojekyll && git add out/ && git commit -m \"Deploy Next.js to gh-pages\" && git subtree push --prefix out origin gh-pages"
+    "deploy": "rm -rf node_modules/.cache && next build && next export && touch out/.nojekyll && git add -f out/ && git commit -m \"Deploy Next.js to gh-pages\" && git subtree push --prefix out origin gh-pages"
```

Comment:
Instead of this change the `.gitignore` should be updated to not include `out`: https://github.com/ronaldstevanus/next.js/blob/fix-gh-pages-deploy-script/examples/gh-pages/.gitignore#L13

Reply:
ok